### PR TITLE
Feature/mar 823

### PR DESCRIPTION
**Pull request by cad106uk**
PR for MAT-823

### CODE REVIEW

File: templates/partials/header.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         </div>
         <ul id="logo-navigation" class="datahub-header__links" aria-label="Header links">
             <li class="datahub-header__links__item">
-                <a class="datahub-header__links__item__text" href="{{ settings.DATAHUB_DOMAIN }}/support">Support</a>
+                <a class="datahub-header__links__item__text" href="https://workspace.trade.gov.uk/"> Switch to Data Workspace </a>
```
Comment (felix781) [inline templates/partials/header.html:16]: @cad106uk the Data Workspace link is https://data.trade.gov.uk/

Comment (jamespacileo) [reply templates/partials/header.html]: @cad106uk any chance we can keep the value coming from an env variable? maybe we need to add something additional to Vault?

**Review by jamespacileo — CHANGES_REQUESTED** [review]
looks good! 1 comment

File: templates/partials/header.html (hunk 1 above)
Comment (cad106uk) [reply templates/partials/header.html]: Should that change be done in this PR? This sounds like something that should be done in a separate PR where we put all the URL domains we could use in all our projects in the Vault. Then make sure all out repos use these values consistently.

Or do we want this one URL to be treated separately here?

Comment (jamespacileo) [reply templates/partials/header.html]: shall we create a backlog ticket? @cad106uk

**Review by jamespacileo — APPROVED** [review]

**Comment by jamespacileo** [comment]
looks good

**Review by felix781 — APPROVED** [review]

**Pull request merged**